lib/browser/rpc-server.js

```diff
@@ -146,6 +146,30 @@ const throwRPCError = function (message) {
   throw error
 }
 
+const rendererMissingErrorMessage = function (meta, args, callIntoRenderer) {
+  let message = `Attempting to call a function in a renderer window that has been closed or released.` +
+    `\nFunction provided here: ${meta.location}`
+
+  if (!args || args.length === 0) return message
+  if (!args[0].sender || !args[0].sender._events) return message
+
+  const eventsAttached = args[0].sender._events
+  const remoteEvents = Object.keys(eventsAttached).filter((eventName) => {
+    return Array.isArray(eventsAttached[eventName])
+      ? eventsAttached[eventName].includes(callIntoRenderer)
+      : eventsAttached[eventName] === callIntoRenderer
+  })
+
+  if (remoteEvents && remoteEvents.length > 0) {
```

Comment:
I don't think `Array.prototype.filter` can return null so I think you can drop the first part of this check and just use `remoteEvent.length > 0`